api/src/database/migrations/20230927A-themes.ts

```diff
@@ -0,0 +1,59 @@
+import type { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+	/**
+	 * Knex doesn't support setting defaults to null (you'll end up with `NULL::character varying`),
+	 * so we'll have to create a new column, copy over the relevant bits, and remove the old
+	 */
+	await knex.schema.alterTable('directus_users', (table) => {
+		table.string('appearance');
+	});
+
+	await knex('directus_users').update({ appearance: 'dark' }).where({ theme: 'dark ' });
+	await knex('directus_users').update({ appearance: 'light' }).where({ theme: 'light ' });
```

Comment:
```suggestion
	await knex('directus_users').update({ appearance: 'dark' }).where({ theme: 'dark' });
	await knex('directus_users').update({ appearance: 'light' }).where({ theme: 'light' });
```
assuming these spaces aren't intended, at least from what I can observe with our existing theme behavior